libmount/src/optmap.c

```diff
@@ -70,56 +70,56 @@
  */
 static const struct libmnt_optmap linux_flags_map[] =
 {
-   { "ro",       MS_RDONLY },                 /* read-only */
+   { "ro",       MS_RDONLY, 0 },              /* read-only */
```

Comment:
Again, why? C standard is pretty obvious about unspecified field initialization.